pages/compare.py

```diff
@@ -610,7 +455,7 @@ def render_card(container, input, caller_id, completions):
           completion,
           concepts=[COMPLETION_CONCEPT],
           metadata={
-              "input_id": inp_input.id,
+              "input_id": inp_input ,
```

Comment:
This is a `job id` we obtain for Uploading text, via SDK not the corresponding `input_id`. We could generate `input_id` and pass to `post_inputs` function and reuse it below

Reply:
Modified code to generate input_id to use.